newIDE/app/src/KeyboardShortcuts/DefaultShortcuts.js

```diff
@@ -39,6 +39,7 @@ const defaultShortcuts: ShortcutMap = {
   OPEN_PROPERTIES_PANEL: 'KeyP',
   TOGGLE_INSTANCES_PANEL: 'KeyI',
   TOGGLE_LAYERS_PANEL: 'KeyL',
+  SCENE_OBJECT_RENAME: 'F2',
```

Comment:
Follow the convention of the other shortcuts by using a verb first:
```suggestion
  RENAME_SCENE_OBJECT: 'F2',
```